logs/lxca_client.log

```diff
@@ -0,0 +1,27 @@
+I, [2017-07-05T04:44:25.176791 #1487]  INFO -- : [XClarityClient::Configuration initialize] - Configuration built successfuly
```

Comment:
Can you remove the log from your commit?  We do not want to push this into the repository.